pkg/satellite/satelliteweb/satelliteql/mutation.go

```diff
@@ -269,24 +269,43 @@ func rootMutation(service *satellite.Service, types Types) *graphql.Object {
 						Type: graphql.NewNonNull(graphql.String),
 					},
 					fieldUserID: &graphql.ArgumentConfig{
-						Type: graphql.NewNonNull(graphql.String),
+						Type: graphql.NewNonNull(graphql.NewList(graphql.String)),
 					},
 				},
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
 					pID, _ := p.Args[fieldProjectID].(string)
-					uID, _ := p.Args[fieldUserID].(string)
+					uID, _ := p.Args[fieldUserID].([]interface{})
 
 					projectID, pErr := uuid.Parse(pID)
-					userID, uErr := uuid.Parse(uID)
 
-					err := utils.CombineErrors(pErr, uErr)
+					var userIDs []*uuid.UUID
+					var userErr error
+
+					for _, userID := range uID {
```

Comment:
In case of an error should it skip the return value or return null in that place? Not sure what the default way to handle is in graphql.

Reply:
Well in case of several uncorrelated errors or some half-completed process it would be descriptive to return result and error. But the problem is that graphql-go library doesn't parse the result if error is returned, so i guess i am forced to return nil if any error occurs